### ISSUE TITLE
Indeterminate State

### ISSUE THREAD

**Issue by 0xcaff** [sethgrid/multibar#13]
Sometimes, you know how long things are going to take. Sometimes you don't. Instead of disabling the progress bar, an indeterminate state would be nice. It would show the elapsed time and input `barProgress(i)`.

**Comment by sethgrid** [comment]
Good idea. I have several, higher priority projects I'm working on currently so I can't give any kind of timeline on when I would look at adding this. I'll leave the issue open and add it to my list of things to do. Cheers!
